samples/.drone.yml

```diff
@@ -48,15 +49,17 @@ steps:
     # "Executing Snakemake workflow."
     - snakemake
 
-    - mkdir "$DRONE_BUILD_NUMBER"
+    #- mkdir "$DRONE_BUILD_NUMBER"
     #- git status | grep modified | awk '{print $2}'
     #- git diff-tree -r --no-commit-id --name-only origin/master . | xargs -I '{}' cp '{}' "$DRONE_BUILD_NUMBER"/
 
     # "[+] Back-Pushing output."
+    - git stash
     - git checkout gin-proc || git checkout -b gin-proc
-    - echo ".snakemake/" > .gitignore
-    - cd "$DRONE_BUILD_NUMBER"/
-    - git status | grep modified | awk '{print $2}' | xargs git add && git status && git commit -am "Build '#'$DRONE_COMMIT_MESSAGE Complete" && git annex sync --content
+    - git checkout master Sample1.txt
```

Comment:
What's `Sample1.txt`?  Is it the desired output file?  If it hasn't been committed to `master` (which it wont be), this will either throw an error or checkout the version that's already in the repository.